### ISSUE TITLE
PHP Warnings in Typo3 11 with PHP 8

### ISSUE THREAD

**Issue by SSFGizmo** [SSFGizmo/direct_mail#1]
(1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Undefined variable $_EXTKEY in /direct_mail/ext_localconf.php line 77

* https://docs.typo3.org/typo3cms/exceptions/master/en-us/Exceptions/1365429656.html

```php
$TYPO3_CONF_VARS['EXTCONF']['direct_mail']['UseImplicitPortToFetch'] = $extConf['UseImplicitPortToFetch'];
```

**Comment by SSFGizmo** [comment]
(1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Undefined array key "feInterface" in /Configuration/TCA/Overrides/fe_users.php line 43

**Comment by SSFGizmo** [comment]
(1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Undefined array key "feInterface" in /Configuration/TCA/Overrides/fe_users.php line 43

**Comment by dmitryd** [comment]
Last two are fixed in b756f0c8657f27a09a2f62d341407ed28ec3e91e

**Comment by SSFGizmo** [comment]
Thx.
